eldritch/eldritch.py

```diff
@@ -189,11 +190,12 @@ def get_modifier(self, thing, attribute):
     for glob in self.globals():
       if not glob:
         continue
-      modifier += glob.get_modifier(thing, attribute)
+      modifier += glob.get_modifier(thing, attribute, self)
     return modifier
 
   def get_override(self, thing, attribute):
-    override = None
+    override = True if attribute.startswith(("can_", "cannot_")) else None
```

Comment:
Or I could put explicit can and cannot attributes (can_seal, can_use, cannot_trade_clues, maybe even can_heal_sanity and _stamina to allow e.g. the HealingStone to ask before using) either here or in BaseAncientOne.